src/main/cpp/src/vcf/genomicsdb_bcf_generator.cc

```diff
@@ -64,6 +64,9 @@ GenomicsDBBCFGenerator::GenomicsDBBCFGenerator(const std::string& loader_config_
     int64_t column_end = contig_info.m_tiledb_column_offset + static_cast<int64_t>(end) - 1; //since VCF positions are 1 based
     m_query_config.set_column_interval_to_query(column_begin, column_end);
   }
+  else if (start != 0 || end != 0) {
+    m_query_config.set_column_interval_to_query(start, end);
+  }
```

Comment:
Are `start` and `end` TileDB column offsets here or contig positions?

Reply:
see above